src/components/presentations/AboutPage/about-page.scss

```diff
@@ -0,0 +1,157 @@
+.title {
+  text-align: center;
+  align-items: center;
+  width: 80%;
+  height: 999px;
+  margin: 0 auto;
+  top: 50px;
+  background: rgba(6, 99, 212, 0.02);
+  border-radius: 50%;
+  font-weight: bold;
+  font-size: 40px;
+}
+
+.top-image {
+  background-image: url(https://res.cloudinary.com/dnch08bzc/image/upload/v1556825503/typewriter.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 50vh;
+}
+
+.top-content {
+  font-weight: bold;
+  font-size: 5vh;
+}
+
+.top-content:last-child {
+  font-weight: normal;
+  font-size: 3vh;
+}
+
+hr {
+  width: 55%;
+}
+
+.card-holder {
+  position: absolute;
+  top: 919px;
+  left: 50px;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 74px;
+  align-items: center;
+}
+
+.text-header {
+  height: 42px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 28px;
+  color: #535353;
+}
+
+.card-holder div p:nth-child(2) {
+color: #000000;
+}
+
+.cards {
+  width: 395px;
+  height: 484px;
+  top: 1050px;
+}
+
+#left {
+  background-image: url(https://res.cloudinary.com/dnch08bzc/image/upload/v1556821938/dna-163466_960_720.jpg);
+}
+#middle {
+  background-image: url(https://res.cloudinary.com/dnch08bzc/image/upload/v1556822016/ilts-science-chemistry-test_119059_large.jpg);
+}
+
+#right {
+  background-image: url(https://res.cloudinary.com/dnch08bzc/image/upload/v1556821917/3d-rendering-human-brain-technology-background_34663-72.jpg);
+}
+
+.bottom-section {
+  border: 1px solid black;
+  width: 100%;
+  height: 658px;
+  background-color: #65aaff;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  padding-top: 195px;
+  margin-top: 300px;
+}
+
+.bottom-image {
+  width: 555px;
+  height: 339px;
+  background-image: url(https://res.cloudinary.com/dnch08bzc/image/upload/v1556821936/39444439254_9566a41625_b.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.text-container {
+  width: 555px;
+  height: 359px;
+}
+
+.text {
+  font-family: 'Roboto', sans-serif;
```

Comment:
This `font-family` declaration could have been done at an upper level of `css`, `index.css` for instance.

Reply:
The `font-family` has already been declared in `index.scss`
<img width="1017" alt="Screen Shot 2019-05-05 at 5 08 27 PM" src="https://user-images.githubusercontent.com/17157020/57196837-99398680-6f58-11e9-8a5f-0ca3cdcff2ad.png">

Reply:
Nice, I've changed the implementation.